index.html

```diff
@@ -294,27 +302,28 @@ <h3>Timeline</h3>
 	<section id="success-criteria">
 	  <h2>Success Criteria</h2>
     <p>
-In order to advance to
-<a href="https://www.w3.org/Consortium/Process//#RecsPR" title="Proposed Recommendation">
-Proposed Recommendation</a>, each normative specification will fulfill the
-<a href="http://www.w3.org/Consortium/Process/#implementation-experience">
-implementation experience required by the W3C Process</a> as follows:
+      In order to advance to
+      <a href="https://www.w3.org/Consortium/Process//#RecsPR" title="Proposed Recommendation">
+      Proposed Recommendation</a>, each normative specification will fulfill the
+      <a href="http://www.w3.org/Consortium/Process/#implementation-experience">
+      implementation experience required by the W3C Process</a> as follows:
     </p>
     <ul>
       <li>
-The Working Group will seek evidence of independent interoperable uses from at
-least two independent implementations for each feature defined in the
-specification.
+        The Working Group will seek evidence of independent interoperable uses from at
+        least two independent implementations for each feature defined in the
+        specification.
       </li>
       <li>
-The group will there are sections in the specifications detailing any known
-security or privacy implications for implementers, Web authors, and end users.
+        The group will there are sections in the specifications detailing any known
+        security or privacy implications for implementers and end users.
```

Comment:
I would rephrase the beginning of the sentence "The group will add sections in the specifications detailing any known security or privacy implications for implementers and end users." so that it becomes a sentence. Then I'd be happy to approve.

Reply:
Oops, @Sakurann, you are right.

Actually... what I tried to do yesterday is to "combine" the previous section with the relevant section text in the template. From hindsight, that was a mistake, it just became more confusing. So I have just committed a version that takes over the boilerplate text from the template verbatim. Could you check that?

@brentzundel can you look at this again, too?

Reply:
@Sakurann sorry, I did not make my wish explicit yesterday; can you give a thumbs up to the latest changes so that we can merge this?

Reply:
Approved! Thank you for fixing, Ivan)